src/ui/public/number_list/number_list.html

```diff
@@ -1,18 +1,21 @@
 <div
   ng-repeat="value in numberListCntr.getList() track by $index"
-  class="form-group vis-editor-agg-form-row vis-editor-agg-form-row">
+  class="form-group vis-editor-agg-form-row vis-editor-agg-form-row"
+>
 
   <input
     ng-model="numberListCntr.getList()[$index]"
     kbn-number-list-input
     input-focus
-    class="form-control">
+    class="form-control"
+  >
 
   <button
     ng-click="numberListCntr.remove($index, 1)"
-    class="btn btn-danger btn-xs"
-    type="button">
-    <i class="fa fa-times"></i>
+    class="kuiButton kuiButton--danger"
```

Comment:
You left out the small here.  Though I actually kinda like it better because its the same height as the input field.  But probably better to be consistent.

<img width="402" alt="screen shot 2017-04-21 at 10 11 16 am" src="https://cloud.githubusercontent.com/assets/16563603/25281298/ef66886a-267a-11e7-95d9-4a2d01171f1a.png">